src/cdomains/intDomain.ml

```diff
@@ -15,6 +15,59 @@ exception Unknown
 exception Error
 exception ArithmeticOnIntegerBot of string
 
+
+
+
+(** Define records that hold mutable variables representing different Configuration values.
+  * These values are used to keep track of whether or not the corresponding Config values are en-/disabled  *)
+  type ana_int_config_values = {
+    mutable interval_threshold_widening : float_precision option;
+    mutable interval_narrow_by_meet : float_precision option;
+    mutable def_exc_widen_by_join : float_precision option;
+    mutable interval_threshold_widening_constants : string option;
+    mutable refinement : string option;
+  }
+  
+  let ana_int_config: ana_int_config_values = {
+    interval_threshold_widening = None;
+    interval_narrow_by_meet = None;
+    def_exc_widen_by_join = None;
+    interval_threshold_widening_constants = None;
+    refinement = None;
+  }
+  
+  let extract_from_option x =
+      match x with
+      | None -> failwith "Config option not read"
+      | Some x -> x
```

Comment:
This is just `Option.get`.

Reply:
Should be fixed in https://github.com/goblint/analyzer/pull/943/commits/f5859c39a057ab937860b07c87eceaa217f4b49c